packages/hs-react-ui/src/components/TextInput/TextInput.tsx

```diff
@@ -194,6 +195,7 @@ const TextInput = ({
         id={id}
         type={type}
         multiLineIsResizable={multiLineIsResizable}
+        {...rest}
```

Comment:
@aVileBroker @wmangimelli What do you think about removing all of the props like `type` as they are implicitly included? 

I'd also recommend renaming `rest` to something more descriptive. Right now, rest refers to all of the React HTML input element props.